src/tests/unit/doughs-serializer.test.js

```diff
@@ -81,25 +81,17 @@ describe('test doughs serializer', () => {
     sinon.restore();
   });
   describe('serializeDough', () => {
-    describe('when it gets a dough object with specialInstructions = `null`', () => {
-      initializeSerializeDoughTest(nullSpecialInstructionsDough);
-
-      itMakesSpecialInstructionsAnEmptyString();
-    });
-
-    describe('when it gets a dough with specialInstructions = \'\'', () => {
-      initializeSerializeDoughTest(emptyStringSpecialInstructionsDough);
-
-      itConvertsIntegerFieldsFromStrings();
-    });
-
-    describe('when it gets a raw dough with a valid format', () => {
+    describe('when it gets a singleton dough object with a valid format', () => {
       initializeSerializeDoughTest(emptyStringSpecialInstructionsDough, 'doughs');
 
       itCreatesTheCorrectTopLevelLink('/v1/doughs');
 
       itConvertsIntegerFieldsFromStrings();
 
+      it('returns `data` as an object', () => {
+        serializedDough.data.should.be.a('object');
```

Comment:
Should be `an` object. Same mistake for the array assertion in `serializeDoughs`.